scripts/deploy_factory_dev.sh

```diff
@@ -0,0 +1,229 @@
+#!/bin/bash
+#### --------------------------------------------
+#### NOTE: The following flows are supported in this file, for testing!
+# - deploy factory v2
+# - create new DAO
+# - create 1-3 proposals
+# - upgrade to factory v3 
+# - submit DAO code v2 
+# - submit DAO metadata v2
+# - submit DAO code v3 
+# - submit DAO metadata v3
+# - upgrade DAO to v3 
+# - check if proposals return
+#### --------------------------------------------
+set -e
+
+if [ -d "res" ]; then
+  echo ""
+else
+  mkdir res
+fi
+
+cd "`dirname $0`"
+
+if [ -z "$KEEP_NAMES" ]; then
+  export RUSTFLAGS='-C link-arg=-s'
+else
+  export RUSTFLAGS=''
+fi
+
+# TODO: Change to the official approved commit:
+COMMIT_V3=596f27a649c5df3310e945a37a41a957492c0322
+# git checkout $COMMIT_V3
+
+# build the things
+cargo build --all --target wasm32-unknown-unknown --release
+cp ../target/wasm32-unknown-unknown/release/*.wasm ../res/
+
+export NEAR_ENV=testnet
+export FACTORY=testnet
+
+if [ -z ${NEAR_ACCT+x} ]; then
+  # export NEAR_ACCT=sputnikv2.$FACTORY
+  export NEAR_ACCT=sputnikpm.$FACTORY
+else
+  export NEAR_ACCT=$NEAR_ACCT
+fi
+
+export FACTORY_ACCOUNT_ID=factory2.$NEAR_ACCT
+# export DAO_ACCOUNT_ID=croncat.sputnikv2.$FACTORY
+export DAO_ACCOUNT_ID=sputnikdao-dev-1644971179.factory2.sputnikpm.testnet
```

Comment:
We should have only 3 constants here: wallet_id, factory_id, dao_id. Anything more than that is just overkill.

Reply:
For dev script its not practical to slim down the constants. it is possible, however it limits being able to step through the process of upgrade. I will leave